__tests__/models.test.js

```diff
@@ -0,0 +1,42 @@
+const request = require("supertest");
+const app = require("../app");
+const testData = require("../db/data/test-data/");
+const seed = require("../db/seeds/seed");
+const db = require("../db/connection");
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => {
+  db.end();
+});
+
+describe("GET /api/categories", () => {
+  test("statuscode: 200", () => {
+    return request(app).get("/api/categories").expect(200);
+  });
```

Comment:
This test isn't necessary - as you're already testing for it in the following test with 'expect(200)'. The second test that you have done should have the expected status code in the description also 😄